packages/onboarding-ui/src/pages/AwaitingConfirmationPage/AwaitingConfirmationPage.tsx

```diff
@@ -50,13 +64,17 @@ const AwaitingConfirmationPage = (): ReactElement => {
             backgroundColor={colors.n700}
             borderRadius='x3'
           >
-            {t('page.awaitingConfirmation.code')}
+            {securityCode}
           </Box>
 
           <Box fontScale='c1'>
             <Trans i18nKey='page.awaitingConfirmation.codeFallback'>
-              <Link href='#' />
-              <Link href='#' />
+              Didn’t receive email?
+              <ActionLink onClick={onResendEmailRequest}>Resend</ActionLink>
+              or
+              <ActionLink onClick={onChangeEmailRequest}>
+                Change email
+              </ActionLink>
```

Comment:
it's necessary to have the text inside of the component?

Reply:
Not really, but without it is hard to guess how it should be structured. It's very different from just replacing a string, when a key is enough.

Reply:
make sense, thanks!